library/src/main/kotlin/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -156,6 +158,11 @@ internal class HttpTransaction(
         requestHeaders = JsonConverter.instance.toJson(headers)
     }
 
+    fun setGraphQlOperationName(headers: Headers) {
+        graphQlOperationName = toHttpHeaderList(headers)
+            .find { it.name.lowercase().contains("operation-name") }?.value
+    }
+
```

Comment:
@cortinico I may have missed something but still curious. Wasn't the plan to let the developers register a GraphQL endpoint in the `ChuckerInterceptor` so that we could avoid identifying GraphQL operations on the basis of certain headers?
I may be wrong but it appears we still may not be able to support GraphQL HTTP GET requests using this change.
Any thoughts?

Reply:
> Though majority of clients use Apollo-Kotlin to execute GraphQL requests but it appears we still may not be able to support GraphQL HTTP GET requests.
Any thoughts?

That's correct. For the time being we're assuming you're having a "operation-name" header. We can expand on this condition in the future to find the Graph QL Operation Name.